docs/execution-environments/index.rst

```diff
@@ -91,10 +91,6 @@ Various Notebook flavours:
 
 .. External links
 
-.. |forum| raw:: html
-
-    <a href="https://community.neptune.ai/c/feature-requests" target="_blank">feature request</a>
-
 .. |neptune-client| raw:: html
 
-    <a href="/api-reference/neptune/index.html" target="_blank">neptune-client</a>
\ No newline at end of file
+    <a href="/api-reference/neptune/index.html" target="_blank">neptune-client</a>
```

Comment:
you can drop this link, as it's not used anywhere.

Reply:
**done**